courseManagementService/src/main/java/com/lms/controller/CourseController.java

```diff
@@ -8,9 +8,15 @@
 import java.util.UUID;
 
 @RestController
+@RequestMapping("/api")
 public class CourseController {
 @Autowired
     private CourseService courseService;
+@GetMapping("/hello")
+public String hello(){
+    System.out.println("hello sir");
```

Comment:
please update your PR if you have any other changes in your local as this is not the required task